setup.py

```diff
@@ -4,9 +4,30 @@
 
 from setuptools import setup
 
+
+if sys.version_info < (3, 6):
+    print("qmt requires Python 3.6 or above.")
+    sys.exit(1)
+
+
+# Loads _version.py module without importing the whole package.
+def get_version_and_cmdclass(package_name):
+    import os
+    from importlib.util import module_from_spec, spec_from_file_location
+
+    spec = spec_from_file_location("version", os.path.join(package_name, "_version.py"))
+    module = module_from_spec(spec)
+    spec.loader.exec_module(module)
+    return module.__version__, module.cmdclass
+
+
+version, cmdclass = get_version_and_cmdclass("qmt")
+
 setup(
     name="qmt",
-    version="1.0",
+    version=version,
+    python_requires=">=3.6",
```

Comment:
`python_requires` will tell to pip ahead of time not to even try to install on incompatible versions